src/main/webapp/app/entities/exercise-hint/exercise-hint-delete-dialog.component.html

```diff
@@ -0,0 +1,20 @@
+<form name="deleteForm" (ngSubmit)="confirmDelete(exerciseHint.id!)">
+    <div class="modal-header">
+        <h4 class="modal-title" jhiTranslate="entity.delete.title">Confirm delete operation</h4>
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true" (click)="clear()">&times;</button>
+    </div>
+    <div class="modal-body">
+        <jhi-alert-error></jhi-alert-error>
+        <p id="jhi-delete-exerciseHint-heading" jhiTranslate="artemisApp.exerciseHint.delete.question" [translateValues]="{ id: exerciseHint.id }">
```

Comment:
I think we should use the title here instead of the id, because no one can remember ids

Reply:
You are right, I'll change it.